calico-enterprise/multicluster/set-up-multi-cluster-management/helm-install/create-a-management-cluster-helm.mdx

```diff
@@ -156,11 +155,52 @@ To install a {{prodname}} [management](create-a-management-cluster-helm#value) c
   watch kubectl get tigerastatus
   ```
 
+
+#### Update the ManagementCluster address.
+
+1. Export the service port number
+
+  ```bash
+  export EXT_NODE_PORT=<your-load-balancer-external-node-port>
+  ```
+
+  Export the public address or host of the management cluster, in this case the load-balancer's external IP (Ex. "example.com:1234" or "10.0.0.10:1234".)
+
+  ```bash
+  export MANAGEMENT_CLUSTER_ADDR=<your-load-balancer-external-addr>:$EXT_NODE_PORT
+  ```
+
+  Replace the `address` field in the ManagementCluster resource.
+    
+  ```bash
+  kubectl patch managementcluster tigera-secure --type merge -p "{\"spec\":{\"address\":\"${MANAGEMENT_CLUSTER_ADDR}\"}}"
+  ``` 
+
 </TabItem>
 <TabItem label="LoadBalancer" value="LoadBalancer">
 
 To install a {{prodname}} [management](create-a-management-cluster-helm#value) cluster with Helm, using a LoadBalancer service:
 
+#### Set-up an EKS LoadBalancer.
+
+  You must meet the requirements defined in [create a network load balancer for AWS](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html)
+
+  Make sure your management cluster has the following annotations:
+
+  ```bash
+  managementCluster:
+    service:
+      annotations:
+      - key: service.beta.kubernetes.io/aws-load-balancer-type
+        value: "external"
+        key: service.beta.kubernetes.io/aws-load-balancer-nlb-target-type
+        value: "instance"
+        key: service.beta.kubernetes.io/aws-load-balancer-scheme
+        value: "internet-facing"
+  ```
```

Comment:
I would suggest that this be moved into step 3 with wording like below to specify EKS

```suggestion
  If you are using EKS, make sure your management cluster has the following annotations:

  ```bash
  managementCluster:
    service:
      annotations:
      - key: service.beta.kubernetes.io/aws-load-balancer-type
        value: "external"
        key: service.beta.kubernetes.io/aws-load-balancer-nlb-target-type
        value: "instance"
        key: service.beta.kubernetes.io/aws-load-balancer-scheme
        value: "internet-facing"
  ```